go.mod

```diff
@@ -1,3 +1,11 @@
 module github.com/TheForgotten69/goinsta/v2
 
 go 1.15
+
+require (
+	github.com/stretchr/testify v1.7.0
+	github.com/tcnksm/go-input v0.0.0-20180404061846-548a7d7a8ee8
+	golang.org/x/crypto v0.0.0-20201221181555-eec23a3978ad // indirect
+	golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c // indirect
+	golang.org/x/term v0.0.0-20201210144234-2321bbc49cbf // indirect
+)
```

Comment:
After thinking about it, It will be much cleaner to add a go mod to each example rather than changing the default go mod for people who won't use these examples in their programs

Reply:
I think it will be even better to remove third-party dependencies from examples